database-mysql/index.js

```diff
@@ -1,16 +1,14 @@
 var mysql = require('mysql');
```

Comment:
Use ES6 `const` or `let` instead of `var`

Reply:
k